packages/pluggable-widgets-tools/CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 -   We updated babel-jest configuration to prevent warnings during unit test execution.
 
+-   We updated the typings generator for icon properties
```

Comment:
You added this under the header of the already released 9.20.0 version, it should go under `[Unreleased]`.

Reply:
✨
It still needs a rebase, but that's a separate concern. :)